tutorials/pl/basic-snap-usage-pl/basic-snap-usage.md

```diff
@@ -0,0 +1,329 @@
+
+
+```bash
+$ snap list
+Name             	Version	Rev  Developer   Notes
+hello            	2.10   	20   canonical   -
+core      	16.04.1	423  canonical   -
+```
+
+Zauważ, że pakiet `core` zawierający podstawowy podsystem również znajduje się na liście. Zawiera on również aktualna wersję snapd, dzięki czemu będzie on zawsze aktualny. 
+
+### Uaktualnianie systemu
+
+Będzie dla Ciebie dobrą wiadomością, że snapy są aktualizowane automatycznie codziennie. Jeżeli chcesz, możesz aktualizować snapy ręcznie poleceniem `snap refresh`. Zaktualizuje to wszystkie Twoje snapy, chyba że określisz konkretny pakiet w poleceniu.
+
+
+```bash
+$ sudo snap refresh hello
+error: cannot refresh "hello": snap "hello" has no updates available
+$ sudo snap refresh
+core updated
+hello 64.75 MB [=====================================>___]   12s
+
+```
+
+Nie są to wszystkie funkcje polecenia snap: możesz bardziej dokładnie śledzić wersje aplikacji, w zależności od tego, jak bardzo chcesz być na bieżąco z nową funkcjonalnością. Jest to możliwe dzięki funkcji kanałów. Możemy zagłębić się w to później.
+
+## Więcej możliwości snapów
+Duration: 1:00
+
+### Snapy mogą dodawać więcej niż jedno polecenie
+
+Nasz pierwszy przykład był prosty i zawierał tylko jedno polecenie, ale zwykle pakiety snap zawierają więcej niż jedno polecenie (na przykład zestaw narzędzi umieszczony w jednym snapie). Wszystkie te polecenia należą do przestrzeni nazw o nazwie pakietu. Wykonaj następujące kroki, aby zobaczyć przykład tego w działaniu:
+
+
+```bash
+$ sudo snap install hello-world
+0 B / 20.00 KB [_______________________________________________________] 0.00 %
+
+hello-world (stable) 6.3 from 'canonical' installed
+$ hello-world
+Hello World!
+$ hello-world.env
+< env variables >
+```
+
+
+positive
+: Informacja: pierwsze polecenie nie posiada przedrostka. To dlatego, że snapy mogą posiadać jedno domyślne polecenie.
+
+### Snap może również zawierać usługi
+
+Czym jest usługa? Jest to uruchomione na dłuższy czas (zwykle działające w tle) polecenie, które prawie zawsze ma na celu szybko odpowiadać na dane żądania.
+Snapy mogą zawierać ogólnosystemowe usługi. Jeżeli zainstalujesz pakiet **shout** (webowy klient IRC), możesz wypróbować tą opcję w działaniu – po prostu przejdź do adresu [http://localhost:9000/] w przeglądarce, aby skorzystać z usługi.
+
+Usługi snapów są uruchamiane/zatrzymywane przy uruchamianiu/wyłączaniu systemu. Mogą zostać skonfigurowane, aby uruchamiały się na żądanie.
+
+### Usuwanie snapów
+
+Usunięcia snapa wymaga tylko jednego polecenia, wszystko co musisz zrobić to `snap remove <snap name>`. W naszym przypadku jest to:
+
+
+```bash
+$ sudo snap remove hello-world
+
+hello-world removed
+```
+
+Łatwo i przejrzyście, nic po nim nie pozostało! Kod aplikacji, jej zależności i dane użytkownika są usuwane. Jeżeli snap zawiera usługę, zostanie ona wyłączona i usunięta.
```

Comment:
This is good but I would really strongly emphasize that user data belonging to that snap is automatically removed.